src/udp_load_balancer.h

```diff
@@ -1,2 +1,25 @@
+#ifndef _DPLB_UDP_LOAD_BALANCER_
+#define _DPLB_UDP_LOAD_BALANCER_
+
+#define udp_load_balancer_version      1000000
+#define UDP_LOAD_BALANCER_VERSION      "1.0.0"
```

Comment:
I think that adding the "version" feature is out of the scope of the PR and should be done in an independent PR (where we can discuss the better place and way to accomplish it)

Reply:
Ok, I just push just another commit to my branch , but it is automatically added to PR. I don't know how to separate it now...